### PR TITLE
Added quarkus modules.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/eventuate-messaging-kafka-quarkus
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - eventuate-messaging-kafka-quarkus-{{ checksum "gradle.properties" }}
```
Comment (cer) [inline .circleci/config.yml]: Please add checksum of wrapper properties

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - run:
+          name: Publish artifacts
+          command: |
+            TERM=dumb ./deploy-artifacts.sh
+          when: on_success
+      - run:
+          name: Save test results
+          command: |
+            mkdir -p ~/junit ~/container-logs
+            docker ps -a > ~/container-logs/containers.txt
```
Comment (cer) [inline .circleci/config.yml]: Isn't there a standard script for this?

File: eventuate-messaging-kafka-quarkus-producer/build.gradle [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    compile 'io.quarkus:quarkus-core'
+    compile 'io.quarkus.arc:arc'
+    compile 'io.quarkus:quarkus-arc-deployment'
+
+    compile "io.eventuate.common:eventuate-common-json-mapper:$eventuateCommonVersion"
+
+    testCompile 'io.quarkus:quarkus-junit5'
+
+    implementation enforcedPlatform("io.quarkus:quarkus-bom:${quarkusVersion}")
```
Comment (cer) [inline eventuate-messaging-kafka-quarkus-producer/build.gradle]: This looks repetitive. Which of these dependencies can be applied in the top-level build.gradle in the `subprojects {...` section.

**Review by cer — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (dartartem) [reply .circleci/config.yml]: ok

File: .circleci/config.yml (hunk 2 above)
Comment (dartartem) [reply .circleci/config.yml]: I will check

File: eventuate-messaging-kafka-quarkus-producer/build.gradle (hunk 3 above)
Comment (dartartem) [reply eventuate-messaging-kafka-quarkus-producer/build.gradle]: I will check

**Pull request merged**